src/anyvar/storage/snowflake.py

```diff
@@ -44,7 +44,9 @@ def __init__(
         snowflake.connector.paramstyle = "qmark"
 
         # get table name override from environment
-        self.table_name = table_name or os.environ.get("ANYVAR_SNOWFLAKE_STORE_TABLE_NAME", "vrs_objects")  # noqa: E501
+        self.table_name = table_name or os.environ.get(
+            "ANYVAR_SNOWFLAKE_STORE_TABLE_NAME", "vrs_objects"
+        )  # noqa: E501
```

Comment:
Don't need this anymore with the reformatting 
```suggestion
        )
```